objcgen/extensions.cs

```diff
@@ -134,4 +134,15 @@ public static void IncrementValue<TKey> (this Dictionary<TKey, int> dictionary,
 				dictionary[key] = new List<TListValue> () { val };
 		}
 	}
+
+	public static class ListExtensions {
+		public static bool ContainsType (this List<ProcessedType> list, Type type)
```

Comment:
I'm fine with it as it helps readability. In fact naming it `Contains` would make the diff a to smaller.